packages/devtools_app/lib/src/http/http_request_data.dart

```diff
@@ -485,9 +485,10 @@ int _dartIoHttpRequestWrapperId = 0;
 
 class DartIOHttpRequestData extends HttpRequestData {
   DartIOHttpRequestData(
-    int? timelineMicrosBase,
+    int timelineMicrosBase,
     this._request,
   )   : wrapperId = _dartIoHttpRequestWrapperId++,
+        _instantEvents = [],
```

Comment:
setting this to the empty list here will prevent _instantEvents from being initialized in the `instantEvents` getter, so let's remove.

Reply:
done